utils/clients/tests/test_busapps_ingestion_client.py

```diff
@@ -0,0 +1,181 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+"""
+Test BusApps ingestion client
+"""
+
+from urllib.error import URLError
+
+import unittest
+
+from mock import patch, MagicMock
+
+from suds import WebFault
+
+from utils.clients.busapps_ingestion_client import BusAppsIngestionClient
+from utils.clients.connection_exception import ConnectionException
+from utils.clients.settings.client_settings_factory import ClientSettingsFactory
+
+
+class TestBusAppsIngestionClient(unittest.TestCase):
+    """
+    Exercises the BusApps ingestion client
+    """
+    def setUp(self):
+        self.valid_value = "valid"
+        self.test_credentials =\
+            ClientSettingsFactory().create('busapps',
+                                           username='valid-user',
+                                           password='valid-pass',
+                                           host='',
+                                           port='',
+                                           uows_url='https://api.valid-uows.com/?wsdl',
+                                           scheduler_url='https://api.valid-scheduler.com/?wsdl')
+
+    def create_client(self, to_mock=None, credentials=None):
+        """ Returns a client with 0 or more mocked instance variables
+        :param to_mock: A list of client instance variables to mock
+        :param credentials: The credentials to initialise the client with
+                            (test credentials used if none supplied)
+        :return: A client with 0 or more mocked instance variables """
+        if not credentials:
+            credentials = self.test_credentials
+        client = BusAppsIngestionClient(credentials)
+        if to_mock and "_uows_client" in to_mock:               # pylint: disable=protected-access
```

Comment:
Should just be able to put the disable for pylint on the top line of these functions and it will do it for all the below

Reply:
This is the best tip ever.
I'm so tired of copying&pasting `pylint:disable` comments, haha